### ISSUE TITLE
add command-line option to assert last point is within control limits

### ISSUE THREAD

**Issue by skuenzli** [skuenzli/time-series-analysis#1]
add a command-line option (--assert-last-point-in-control) to assert last point is within control limits

**Comment by skuenzli** [comment]
return a non-zero exit status when --assert-last-point-in-control is present and last point it is out of control (ooc)

**Comment by skuenzli** [comment]
there don't appear to be hard-and-fast standards for exit codes, but looks like there are some things we can go with the flow on:
- http://stackoverflow.com/questions/1101957/are-there-any-standard-exit-status-codes-in-linux
- http://tldp.org/LDP/abs/html/exitcodes.html
- perhaps the best guidance is from /usr/include/sysexits.h (on linux & osx):

``` c
#define EX_OK           0       /* successful termination */

#define EX__BASE        64      /* base value for error messages */

#define EX_USAGE        64      /* command line usage error */
#define EX_DATAERR      65      /* data format error */
#define EX_NOINPUT      66      /* cannot open input */
#define EX_NOUSER       67      /* addressee unknown */
#define EX_NOHOST       68      /* host name unknown */
#define EX_UNAVAILABLE  69      /* service unavailable */
#define EX_SOFTWARE     70      /* internal software error */
#define EX_OSERR        71      /* system error (e.g., can't fork) */
#define EX_OSFILE       72      /* critical OS file missing */
#define EX_CANTCREAT    73      /* can't create (user) output file */
#define EX_IOERR        74      /* input/output error */
#define EX_TEMPFAIL     75      /* temp failure; user is invited to retry */
#define EX_PROTOCOL     76      /* remote error in protocol */
#define EX_NOPERM       77      /* permission denied */
#define EX_CONFIG       78      /* configuration error */

#define EX__MAX 78      /* maximum listed value */

```

**Comment by skuenzli** [comment]
Closed by 1f5f734
